packages/react-components/src/localization/locales/en-US/strings.json

```diff
@@ -133,7 +133,9 @@
     "callCameraAlreadyInUse": "Unable to access camera. It may already be in use by another application.",
     "callMacOsCameraAccessDenied": "MacOS is blocking access to your camera. Update your privacy settings to allow this browser to access your camera.",
     "callMacOsScreenShareAccessDenied": "MacOS is blocking screen sharing. Update your privacy settings to allow this browser to record your screen.",
-    "dismissButtonAriaLabel": "Close"
+    "dismissButtonAriaLabel": "Close",
+    "failedToJoinCallGeneric": "Failed to join call.",
+    "failedToJoinCallInvalidMeetingLink": "Unable to join Meeting. Invalid Link."
```

Comment:
What's the reasoning behind using `call` and `Meeting` for separate errors.

Reply:
I used "Meeting" over call in one of them as this error is specific for invalid Teams Meetings links (we could update this string for Teams Meeting).

Then there's two strings here:
1. Failed to join from an invalid teams link -- this felt actionable by the end user as often meetings can't be joined because someone was sent or copy pasted the link incorrectly.
2. Otherwise we just have a callAgent.Join error which is very generic catch-all error message as this could be due to anything so unless they specifically can't join because of an invalid teams link we just show "Failed to join call."